### ISSUE TITLE
Button accessible name context

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag3#184]
Back in 2016 we deferred resolution "button context" in https://github.com/w3c/wcag/issues/168 to WCAG Next..

If we had a piece of text such as a person's name and then an Add button to the right -- do people agree that this is a failure of SC 1.3.1 or a pass?  The Add button itself might say "Add" but doesn't say what it adds.  Visually there is a relationship but not  programmatically.   SC 2.4.6 only seems to address the label "add" -- which seems like it is clear visually in combination -- but it's the accessible name that is more problematic.    Adding "Add Persons's name" to the button would add extra text and then it would be best to either merge the button and text into one control or hide the text from screen readers in order to prevent extra verbosity.    

The authors of WCAG 2 felt that 2.4.4 was needed while 1.3.1 could have solved the issue of link context-- so does that mean button context is not covered by 1.3.1 or 2.4.6 either?   Buttons generally aren't in the same sentence or paragraph like links can be and button's may apply to groups of controls like a "submit" button that actually submits all fields in a form rather than just the field before it.  So where do we draw the lines for buttons?
